inst/doctest.m

```diff
@@ -165,8 +165,8 @@
 %%
 %% @example
 %% @group
-%% >> "shiny Octave feature"    % doctest: +XFAIL_IF(DOCTEST_MATLAB)
-%% ans = shiny Octave feature
+%% >> license       % doctest: +XFAIL_IF(DOCTEST_MATLAB)
+%% ans = GNU General Public License
```

Comment:
This is indeed the shiniest possible feature any software can have :-)

This must make @mtmiller a contender for greatest short commit ever!